Dockerfile

```diff
@@ -1,39 +1,49 @@
-FROM golang AS aws-k8s-tester-builder
+ARG GO_VERSION
+ARG AL_VERSION
+ARG RELEASE_VERSION
+ARG OS_TARGET
+ARG OS_ARCH
+
+FROM golang:${GO_VERSION} AS aws-k8s-tester-builder
 RUN go version
 ADD ./ /go/src/github.com/aws/aws-k8s-tester
 WORKDIR /go/src/github.com/aws/aws-k8s-tester
-ARG RELEASE_VERSION=latest
-ARG OS_TARGETS=linux
-RUN ./hack/build.sh
+RUN GOPROXY=direct ./hack/build.sh
```

Comment:
We should avoid hard-coding `GOPROXY=direct`. Can you pass `GOPROXY` as a `--build-arg` if it's defined?